packages/react-components/react-progress/Spec.md

```diff
@@ -2,62 +2,105 @@
 
 ## Background
 
-_Description and use cases of this component_
+A `Progress` is a component that is used to display the current progress of an operation flow.
 
 ## Prior Art
 
-_Include background research done for this component_
+### Open UI
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+| Library    | Component Name | Spec Link                                                          | Notes                                                                   |
+| ---------- | -------------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------- |
+| Ant Design | Progress       | [Progress](https://ant.design/components/progress/)                | Specifies the type, a combined progress component                       |
+| Bootstrap  | Progress       | [Progress](https://getbootstrap.com/docs/4.3/components/progress/) | Allows for multiple different animation styles on the same Progress bar |
```

Comment:
I am surprised with such small list of prior art as this component is quite popular:
- https://react.semantic-ui.com/modules/progress/
- https://react-spectrum.adobe.com/react-spectrum/ProgressBar.html
- https://vuetifyjs.com/en/components/progress-linear/
- https://mui.com/material-ui/react-progress/

Did you consider other libraries in a research?